From 17e62d5673b9fbc62c09e2b3e7679c07bde85d88 Mon Sep 17 00:00:00 2001
From: Varadhan M <varadhan@cem-solutions.net>
Date: Wed, 11 Jul 2018 05:07:01 -0700
Subject: Fixed: Fraud detection allow leap min

---
 modules/fraud_detection/frd_load.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/modules/fraud_detection/frd_load.c b/modules/fraud_detection/frd_load.c
index fe70a600d..268bcc978 100644
--- a/modules/fraud_detection/frd_load.c
+++ b/modules/fraud_detection/frd_load.c
@@ -115,7 +115,8 @@ static int strtime(const str *time, int *ihrs, int *imin)
 		goto parse_error;
 
 	if (uhrs > 23 || umin >= 60)
-		goto parse_error;
+		if(!(uhrs == 23 && umin == 60)) /* allow the leap min 23:60 */
+			goto parse_error;
 
 	*imin = umin;
 	*ihrs = uhrs;
-- 
2.17.GIT

