From f0ea2b91709bb5fc9de613addb24cec7458fad69 Mon Sep 17 00:00:00 2001
From: Varadhan M <varadhan@cem-solutions.net>
Date: Wed, 22 Jun 2016 03:04:55 -0700
Subject: [PATCH] Removing the Refer-To encoding not required

---
 modules/dialog/dlg_tophiding.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/modules/dialog/dlg_tophiding.c b/modules/dialog/dlg_tophiding.c
index 9fd1f30..a56821e 100644
--- a/modules/dialog/dlg_tophiding.c
+++ b/modules/dialog/dlg_tophiding.c
@@ -1379,10 +1379,12 @@ int dlg_th_pre_raw(str *data, struct sip_msg* foo)
 					LM_ERR("Failed to decode callid for sequential request\n");
 					goto error;
 				}
+#if 0
 				/* encoding REFER-TO shouldn't be by default */
 				if(!dlg_th_needs_referto_decoding(&msg)) { /* If not already encoded */
 					dlg_th_encode_referto(&msg);
 				}
+#endif
 				goto rebuild_msg;
 			}
 		} else {
-- 
1.7.1

